src/adapters/jquery.js

```diff
@@ -1,14 +1,12 @@
-(function() {
   'use strict'
 
-  var $ = window.jQuery
-  var Waypoint = window.Waypoint
+  var jquery = global.jQuery  //require('jquery')
```

Comment:
Is the comment here a remnant of an attempt to load jQuery via require, or a reminder to change it to require later at some point, or something else entirely?